bot.py

```diff
@@ -28,111 +29,27 @@
 
 startTime = 0
 
-bot = discord.Client(intents=intents)
+bot = commands.Bot(command_prefix="/", intents=intents)
+
 
-tree = app_commands.CommandTree(bot)
 
 @bot.event 
 async def on_ready():  # When the bot is ready
-    tree.add_command(Bot())
-    tree.add_command(Misc())
-    tree.add_command(Minecraft())
-    tree.add_command(Web())
 
-    await tree.sync()  # Syncs the command tree
+    bot.tree.add_command(Bot())
+    bot.tree.add_command(Misc())
+    bot.tree.add_command(Minecraft())
+    await bot.load_extension("cogs.web")
+
+    await bot.tree.sync()  # Syncs the command tree
 
     print("Ready!")
     await bot.change_presence(activity=discord.Activity(type=discord.ActivityType.watching, name="over your bots!"))
     print(bot.user)  # Prints the bot's username and identifier
 
-    # Unused variable, why?
-    #startTime = time()
```

Comment:
This is so that a !uptime command (or similar) can be added that I haven't implemented. I'm like 99% sure it's in V2

Reply:
I didn't write that comment lol